src/react-components/ui-root.js

```diff
@@ -1889,17 +1889,11 @@ class UIRoot extends Component {
                   onEndShareVideo={this.endShareVideo}
                   onShareVideoNotCapable={() => this.showWebRTCScreenshareUnsupportedDialog()}
                 />
-                {this.props.isSupportAvailable && (
```

Comment:
Is this feature deprecated and if so, has the rest of the code supporting it been removed?

Reply:
still TBD, I'll remove it in a week or so after this goes live if we see this new mechanism actually working -- good chance we will revisit if not

Reply:
fwiw we did actually remove the polling, so its already disabled